extension/src/plots/model/custom.ts

```diff
@@ -16,28 +18,93 @@ export const isCheckpointValue = (
 export const isCheckpointPlot = (plot: CustomPlot): plot is CheckpointPlot =>
   plot.type === CustomPlotType.CHECKPOINT
 
-export const removeColumnTypeFromPath = (
-  columnPath: string,
-  type: string,
-  fileSep: string
-) =>
-  columnPath.startsWith(type + fileSep)
+export const removeColumnTypeFromPath = (columnPath: string, type: string) =>
+  columnPath.startsWith(type + FILE_SEPARATOR)
     ? columnPath.slice(type.length + 1)
     : columnPath
 
-export const getFullValuePath = (
-  type: string,
-  columnPath: string,
-  fileSep: string
-) => type + fileSep + columnPath
+export const getFullValuePath = (type: string, columnPath: string) =>
+  type + FILE_SEPARATOR + columnPath
 
 export const cleanupOldOrderValue = (
-  value: { metric: string; param: string } | CustomPlotsOrderValue,
-  fileSep: string
+  value: { metric: string; param: string } | CustomPlotsOrderValue
 ): CustomPlotsOrderValue => ({
   // previous column paths have the "TYPE:" prefix
-  metric: removeColumnTypeFromPath(value.metric, ColumnType.METRICS, fileSep),
-  param: removeColumnTypeFromPath(value.param, ColumnType.PARAMS, fileSep),
+  metric: removeColumnTypeFromPath(value.metric, ColumnType.METRICS),
+  param: removeColumnTypeFromPath(value.param, ColumnType.PARAMS),
   // previous values didn't have a type
   type: (value as CustomPlotsOrderValue).type || CustomPlotType.METRIC_VS_PARAM
 })
+
+export const getCustomPlotIds = (
+  customPlotVals: CustomPlotsOrderValue[],
+  customPlotType: CustomPlotType
+): Set<string> => {
+  const plotIds: Set<string> = new Set()
+
+  for (const { type, metric, param } of customPlotVals) {
+    if (type === customPlotType) {
+      plotIds.add(getCustomPlotId(metric, param))
+    }
+  }
+
+  return plotIds
+}
+
+export const getCustomPlotPathsFromColumns = (
```

Comment:
This function and the one above are the same as before, just moved from `quickPick` to `custom`.